assets/app/scripts/services/storage.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+angular.module("openshiftConsole")
+  .factory("StorageService", function() {
+    function StorageService() {}
+
+    StorageService.prototype.generateSimpleName = function(prefix, length) {
+      if (!prefix) {
+        prefix = "";
+      }
+      if (!length) {
+        length = 5;
+      }
+      var randomString = Math.round((Math.pow(36, length + 1) - Math.random() * Math.pow(36, length))).toString(36).slice(1);
+      return prefix + randomString;
```

Comment:
We have similar code in `ApplicationGenerator._generateSecret`. Is it possible to reuse that?